scripts/configs/UnrollReduce.conf

```diff
@@ -0,0 +1,43 @@
+[General]
+Expression = stencil1D
+InputSize = 1024
+Name = stencil
+
+[HighLevelRewrite]
+Depth = 6
+Distance = 8
+ExplorationDepth = 6
+Repetition = 1
+Collection = convolution1D
+OnlyLower = false
+
+[MemoryMappingRewrite]
+UnrollReduce = true
+Global0 = true
+Global01 = false
+Global10 = false
+Global012 =false
+Global210 = false
+Group0 = false
+Group01 = false
+Group10 = false
+
+[ParameterRewrite]
+DisableNDRangeInjection = true
+Sequential = false
+Settings = stencil1D.json
+InjectNDRange = false
+ExploreNDRange = false
+# everything below here requires 'Explore = true'
+SampleNDRange = 10
+
+[Harness]
+Name = harness_stencil
+Args = -s 1024
+
+[ATF]
```

Comment:
no, not for this pull-request, lets delete it here but add the atf functionality in a seperate pull-request since i'd like to try this for my benchmarks too